docs/ios/v2/how-to-guides/interact-with-room/room/polls.mdx

```diff
@@ -0,0 +1,155 @@
+---
+ title: Polls
+ nav: 7.26
+---
+
+## Introduction
```

Comment:
can remove this heading since the page auto-generates a heading

Reply:
done